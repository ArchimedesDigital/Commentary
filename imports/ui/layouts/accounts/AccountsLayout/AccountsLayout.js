import React from 'react';
import PropTypes from 'prop-types';

class AccountsLayout extends React.Component {
	render() {
		return (
			<main>
				{this.props.content}
			</main>
		);
	}
}

AccountsLayout.propTypes = {
	content: PropTypes.object,
};

export default AccountsLayout;
