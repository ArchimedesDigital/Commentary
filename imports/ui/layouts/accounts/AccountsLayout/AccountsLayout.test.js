import React from 'react';
import renderer from 'react-test-renderer';

// component:
import AccountsLayout from './AccountsLayout';

describe('AccountsLayout', () => {
	it('renders correctly', () => {

		const tree = renderer
			.create(<AccountsLayout />)
			.toJSON();
		expect(tree).toMatchSnapshot();
	});
});
