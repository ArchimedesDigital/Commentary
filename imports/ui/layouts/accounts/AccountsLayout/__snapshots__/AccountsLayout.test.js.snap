// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AccountsLayout renders correctly 1`] = `<main />`;
